packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -682,6 +686,99 @@ class Viewport implements IViewport {
     return true;
   }
 
+  /**
+   * Resets the stored initial offset values for
+   * zoom and pan.  These values are the offset
+   * values used to allow the getPan and getZoom
+   * to return [0,0] and 1 for the initial values.
+   * @param resetOffsets can be passed to skip resetting, ie a no-op on this call
+   */
+  protected resetInitialOffsets(resetOffsets = true) {
+    if (!resetOffsets) {
+      return;
+    }
+    this.initialCamera = this.getCamera();
+  }
+
+  /**
+   * Helper function to return the current canvas pan value.
+   *
+   * @returns a Point2 containing the current pan values
+   * on the canvas,
+   * computed from the current camera, where the initial pan
+   * value is [0,0].
+   */
+  public getPan(): Point2 {
+    const activeCamera = this.getVtkActiveCamera();
+    const focalPoint = activeCamera.getFocalPoint() as Point3;
+
+    const zero3 = this.canvasToWorld([0, 0]);
+    const initialCanvasFocal = this.worldToCanvas(
+      <Point3>vec3.subtract(vec3.create(), this.initialCamera.focalPoint, zero3)
+    );
+    const currentCanvasFocal = this.worldToCanvas(
+      <Point3>vec3.subtract(vec3.create(), focalPoint, zero3)
+    );
+    const result = <Point2>(
+      vec2.subtract(vec2.create(), initialCanvasFocal, currentCanvasFocal)
+    );
+    return result;
+  }
+
+  /**
+   * Sets the canvas pan value relative to the initial view position of 0,0
+   * Modifies the camera to perform the pan.
+   */
+  public setPan(pan: Point2, resetOffsets = false) {
```

Comment:
again `resetOffsets` is confusing to me, I don't understand what it does. + function documentation miss arguments

Reply:
Renamed to storeInitialCamera